js/deer-utils.js

```diff
@@ -105,6 +115,23 @@ export default {
             return label || noLabel
         }
     },
+    postView(entity, matchOn = ["__rerum.generatedBy", "creator"]){
```

Comment:
This is the model for form handling and other messages sent into the worker as well.